schema/partner_show.js

```diff
@@ -252,7 +252,11 @@ const PartnerShow = {
   },
   resolve: (root, { id }) => {
     return gravity(`show/${id}`).then(show => {
-      if (!show.displayable) return new Error("Show Not Found")
+      if (!show.displayable) {
+        const error = new Error("Show Not Found")
```

Comment:
Just a general comment - would be nice to use something more specific than `Error`. Maybe we do this all over MP but it's not great practice.

Reply:
I agree, in principle. Last night I looked into that, but quickly realised that, as is so often the case, the JS community has many opinions on the matter, not simplified by the prototypical vs class based inheritance debates. Many of them revert to duck-typing.

I took another stab at it and have pushed a working version, but I decided to still check for the existence of the property to identify the error in our error handler and not e.g. `instanceof`.